packages/flutter_goldens/test/flutter_goldens_test.dart

```diff
@@ -481,6 +481,23 @@ void main() {
         );
       });
 
+      test('asserts .png format', () async {
```

Comment:
Maybe rewrite these with the `throwsA` type matcher using `having` similarly to how it is done in: https://github.com/flutter/flutter/blob/ab89ce285f76a3fd3a9328ec863f5623cc5ac8bc/packages/flutter/test/gestures/monodrag_test.dart#L60. Unlike that example, you may have to use `await expectLater` since the call is async here.

In general, I find that style less error prone (because the `fail` call is easy to forget) and a little clearer in its intention.

Reply:
Ah! Thank you! I was patting myself on the back fro remembering the `fail`, while simultaneously trying to remember the better way to do it. 😜 
Will update.